### DISCUSSION TITLE
🏆🏆 Vote Directus for the Critic CMS Awards!

### DISCUSSION THREAD

**Discussion by estheragbaje** [directus/directus#21434] [Team Posts]
Hi Community,

Thank you for the nominations! We're thrilled to announce that Directus was nominated for eight (8) categories in the 2023 CMS Critic Awards!!! Now, we need your help to bring these awards home. 

Please take a few minutes to vote for Directus in the final stage of this contest. Every single vote in each category counts and your support means the world to us 💜

👉🏽 To vote, follow this link: https://cmscritic.com/vote 👈🏽

Voting closes on Monday, February 26th! With your help, we can make this happen. 

Thanks in advance for your support! 😊

**Comment by dearmosin** [comment]
I voted for Directus
